examples/step-67/step-67.cc

```diff
@@ -1498,7 +1504,7 @@ namespace Euler_DG
           const Number bi = factor_solution;
           if (ai == Number())
             {
-              DEAL_II_OPENMP_SIMD_PRAGMA
+              // DEAL_II_OPENMP_SIMD_PRAGMA
```

Comment:
This is going to look strange in doxygen because it's rendered as text rather than as a comment. Use `/* ... */` instead.

Reply:
Agreed. Updated.